src/components/Checkbox/BaseCheckbox.js

```diff
@@ -93,6 +93,12 @@ class BaseCheckbox extends React.Component {
         if (event.type && event.type !== 'click') {
             return;
         }
+        const wasChecked = this.props.isChecked;
+
+        // // Make sure checkbox get unfocused when it's unselected
```

Comment:
```suggestion
        // Make sure the checkbox gets unfocused when it's about to be unselected
```

Please pay attention to the changes you are making.